doc/setup.md

```diff
@@ -39,14 +39,14 @@ cd genalyzer
 
 ```bash
 sudo apt-get install python3-dev python3-pip
-sudo pip3 install numpy
+sudo pip3 install -r requirements_test.txt
 ```
 ````
 
 ````{tab} macOS
 
 ```bash
-pip3 install numpy
+pip3 install -r requirements_test.txt
```

Comment:
These should not be changed. As a user you only need numpy

Reply:
Updated.